src/NUnitEngine/EngineApiVersion.cs

```diff
@@ -4,5 +4,5 @@
 
 [assembly: AssemblyProduct("NUnit Engine API")]
 [assembly: AssemblyVersion("3.0.0.0")]
-[assembly: AssemblyInformationalVersion("3.13.0")]
+[assembly: AssemblyInformationalVersion("4.0.0")]
 [assembly: AssemblyFileVersion("3.13.0")]
```

Comment:
Think you've missed a version here. 🙂

Reply:
I think there's also a version stored in the Cake file, which should be updated in sync with these?

Reply:
Your right that I missed one. I'll fix.

However, the cake file was updated when I created dev-4.0 so doesn't need to be changed here.